app/interfaces/api/entities/search_result.rb

```diff
@@ -60,31 +60,31 @@ def redetermination
       end
 
       def fixed_fee
-        object.is_fixed_fee.eql?('t')
+        object.is_fixed_fee.eql?('t') && is_submitted?
```

Comment:
i assume there is some mapping to the case_type.is_fixed_fee attribute going on here. Is there no `def fixed_fee?` method that can be used?